### PR TITLE
DP-45: Encryption

### PR DESCRIPTION
**Pull request by acarlton**
Encryption tweaks -- research report in JIRA ticket

### CODE REVIEW

File: cdk/drupal/scripts/app_launch_config_user_data.sh [hunk 1]
```diff
@@ -135,6 +135,12 @@ echo "${AppEfs}:/ /mnt/efs efs _netdev 0 0" >> /etc/fstab
 mkdir -p /mnt/efs/drupal/files
 chown www-data /mnt/efs/drupal/files
 
+# rds
+mkdir -p /opt/aws/rds
+cd /opt/aws/rds
+wget https://www.amazontrust.com/repository/AmazonRootCA1.pem
+cd -
+
```
Comment (dylanvaughn) [inline cdk/drupal/scripts/app_launch_config_user_data.sh:143]: I see you are also doing this in the AMI (which seems like the right place) - should we remove this?

File: cdk/drupal/drupal_stack.py [hunk 2]
```diff
@@ -25,19 +25,21 @@
...
+# 3) $ ave oe-patterns-dev make AMI_ID=ami-fromstep2 copy-image
 # 4) Copy the code that copy-image generates below
 
 # AMI list generated by:
-# make AMI_ID=ami-062b13bbef7bad514 copy-image
-# on Tue May 26 21:56:04 UTC 2020.
-AMI_ID="ami-062b13bbef7bad514"
-AMI_NAME="ordinary-experts-patterns-drupal-0.1.0-85-gdddc067-20200526-0934"
+# make AMI_ID=ami-06bfd9042474cab3f copy-image
+# on Wed May 27 13:59:02 UTC 2020.
```
Comment (dylanvaughn) [inline cdk/drupal/drupal_stack.py:37]: can you check you have the latest from develop and run the `copy-image` again and re-do the output part? I changed it.

File: cdk/drupal/scripts/app_launch_config_user_data.sh (hunk 1 above)
Comment (acarlton) [reply cdk/drupal/scripts/app_launch_config_user_data.sh]: Removed.

File: cdk/drupal/drupal_stack.py (hunk 2 above)
Comment (acarlton) [reply cdk/drupal/drupal_stack.py]: @dylanvaughn just updated the ami again with develop

**Comment by acarlton** [comment]
@dylanvaughn I addressed your comments -- merge when ready

**Pull request merged**